packages/waku/src/lib/plugins/vite-plugin-rsc-transform.ts

```diff
@@ -223,40 +238,46 @@ const transformServer = (
     }
   }
   if (hasUseClient) {
-    const exportNames = collectExportNames(mod);
+    const { exports, internal } = collectExportNames(mod);
     let newCode = `
 import { registerClientReference } from 'react-server-dom-webpack/server';
 `;
-    for (const name of exportNames) {
+    for (const name of [...exports, ...internal]) {
       newCode += `
 export ${name === 'default' ? name : `const ${name} =`} registerClientReference(() => { throw new Error('It is not possible to invoke a client function from the server: ${getClientId(id)}#${name}'); }, '${getClientId(id)}', '${name}');
 `;
     }
     return newCode;
-  } else if (hasUseServer) {
```

Comment:
I'm not sure if we can remove this branch. It's for another use case unless I'm mistaken.